### PR TITLE
GEODE-8428: Update geode-redis README

### PR DESCRIPTION
**Pull request by sabbeyPivotal**
Update geode-redis README with a short explanation of how to run a Geode server that accepts Redis commands.

### CODE REVIEW

File: geode-redis/README.md [hunk 1]
```diff
@@ -1,78 +1,302 @@
...
+    * --redis-bind-address: This will be the address your client uses to connect. 
+
+    Your Geode instance should now be up and running (1 locator and 1 server) and ready to accept Redis 
+    commands.  
+
+    **Keep this terminal open and running so that you can easily shutdown the Geode instance when you are 
+    done working locally.**
+
+3. To confirm that things are running correctly, in a separate terminal run:
+      ```commandline
```
Comment (dschneider-pivotal) [inline geode-redis/README.md:86]: Should we say something here about the version of redis they should use and how to get the redis-cli installed?

File: geode-redis/README.md [hunk 2]
```diff
@@ -1,78 +1,302 @@
...
+2. After the locator has started, start a server that will be able to handle incoming Redis commands. 
 
-###The `shacompare.sh` Script
-The `shacompare.sh` script is used to compare performance between different commits. This script
-takes in two different commit hashes, checks them out, uses `environment-setup.sh` script to start
-up the Geode Redis server, and runs the benchmarks for each commit. It generates two summary CSV
-files, labeled with the short commit hash, that can be compared for performance changes.
+    For example:
+    ```commandline
+    gfsh> start server --name=redisServer1 --locators=localhost[10334] --server-port=0 --redis-port=6379 --redis-bind-address=127.0.0.1
```
Comment (dschneider-pivotal) [inline geode-redis/README.md:71]: I think, for this example, the --redis-bind-address is not needed (we listen on all local addresses by default).

Comment (sabbeyPivotal) [reply geode-redis/README.md]: good call, just removed it

File: geode-redis/README.md (hunk 1 above)
Comment (sabbeyPivotal) [reply geode-redis/README.md:85]: I added a subtle version mention to the intro paragraph, let me know what you think!

**Pull request merged**